rust/stackablectl/src/cmds/demo.rs

```diff
@@ -278,6 +316,7 @@ async fn install_cmd(
     cli: &Cli,
     list: demo::List,
     transfer_client: &xfer::Client,
+    release_branch: &str,
 ) -> Result<String, CmdError> {
     info!("Installing demo {}", args.demo_name);
```

Comment:
suggestion: We should add the release branch as a field here.

```suggestion
    info!(%release_branch, "Installing demo {}", args.demo_name);
```